### PR TITLE
Short hex colors, e.g. #fff

### PR DESCRIPTION
**Pull request by ChayimFriedman2**

### CODE REVIEW

File: src/modules/math.wren [hunk 1]
```diff
@@ -135,27 +135,21 @@ var NumToHex = Fn.new {|num|
...
-  } else {
-    Fiber.abort("Invalid hex")
-  }
-  if (48 <= second && second <= 57) {
-    second = second - 48
-  } else if (65 <= second && second <= 70) {
-    second = 10 + second - 65
-  } else if (97 <= second && second <= 102) {
-    second = 10 + second - 97
+var ToNum = Fn.new {|digit|
```
Comment (avivbeeri) [inline src/modules/math.wren:138]: We might want to rename this "CharToNum" or "ByteToNum" just to be clear what its purpose is.

**Review by avivbeeri — CHANGES_REQUESTED** [review]
Tiny change, but otherwise looks good.

File: src/modules/math.wren (hunk 1 above)
Comment (ChayimFriedman2) [reply src/modules/math.wren:138]: I followed the convention of `NumToHex` -> `ToHex`, hence `HexToNum` -> `ToNum`.

Comment (ChayimFriedman2) [reply src/modules/math.wren:138]: But `CharToNum` really seems reasonable.
I'll change that to `HexDigitToNum`.

Comment (avivbeeri) [reply src/modules/math.wren]: I missed that connection, apologies. 

We might as well rename the `toHex` as well, to be `NumToHexDigit` or similar, for symmetrical clarity.

Comment (ChayimFriedman2) [reply src/modules/math.wren]: Yes but I don't want to touch the existing code in this PR

Comment (ChayimFriedman2) [reply src/modules/math.wren]: If you want I can do that of course, though

**Comment by avivbeeri** [comment]
Oops, I was trying to be smart and helpful and sort of messed up this PR's commits a bit. My apologies.

**Comment by avivbeeri** [comment]
I thought I'd just add on the method renames, and one other change, but I ended up checking out and merging in changes from a newer version of `develop` than you must have used. For some reason. github seems to think those commits are new in this PR, even though they aren't, which is muddying things up a bit.

**Comment by avivbeeri** [comment]
Ah, I seem to have resolved it!

**Review by avivbeeri — APPROVED** [review]

**Comment by avivbeeri** [comment]
Extending the scope of this PR a bit, but it'd be good for the hexcodes to support optional alpha channels, which shouldnt be too much of an issue.

**Comment by ChayimFriedman2** [comment]
Yes I also thought about that. I'll do that in the near future.

**Comment by ChayimFriedman2** [comment]
Done.

**Comment by avivbeeri** [comment]
Fantastic. Last thing, feel free to add yourself to the contributors list in CONTRIBUTING.md, with a link to your stuff :)

**Comment by ChayimFriedman2** [comment]
This is the thing that I always forget to do 😉

**Pull request merged**